### DISCUSSION TITLE
Shared storage usage

### DISCUSSION THREAD

**Discussion by kevinnashuk** [community/community#46291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I had an email about my "Shared storage usage", "465MB of 512.0MB" is this the size of my stored repositories?  On the web UI, I cannot find any reference to the size of each of the repositories.

Then is says "You've used 90% of included services for GitHub Storage (GitHub Actions and Packages). "

I am confused, is this a file storage warning, or a usage allowance warning, or both?

All help much appreciated!

Thanks

**Comment by airtower-luna** [comment]
From the message you quote it should be about artifacts created from Actions workflows, and packages you might have uploaded. There's [no hard limit](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits) for the repositories themselves.

**Reply by 424778940z** [reply]
same issue here, so where I can ACTUALLY find out WHICH repo is taking up space????

**Comment by grantc** [comment]
> same issue here, so where I can ACTUALLY find out WHICH repo is taking up space????

Under billing, run a report. They will send you a CSV in which you can see which repos are using shared storage.
